Lecture_3/L3_ex3.c

```diff
@@ -0,0 +1,63 @@
+/******************************************************************************
+
+Create linked list library with functions (ListAdd(), ListIs(), ListRemove()).
+New elements in list should be dynamically allocated.
+
+
+*******************************************************************************/
+#include <stdio.h>
+#include <stdlib.h>
+#include "ListLib.h"
+
+
+
+int main()
+{
+    link_list_t *list_init = NULL;
+    link_list_t *for_debug = NULL;
+    int index_in_list;
+    
+    list_init = malloc(sizeof(link_list_t));
+    
+    if (list_init == NULL)
+    {
+        printf("Malloc error\n");
+        return -1;
+    }
+    list_init -> number = 1;
+    list_init -> digit = 0.5;
+    list_init -> next_element = NULL;
+    
+    for (int i = 0; i < 10; i++)
+    {
+        ListAdd (list_init, rand()%10, rand()%10/1.0);
+        
+    }
+    printf("\n");
+    
+    /* * * * * * debug * * * * * * * * * * * * * * * * * *
```

Comment:
If you need to hide some debug stuff it is better to use the following construction:
#ifdef DEBUG
/* some implementation */
#endif